### ISSUE TITLE
Det er ikke muligt at definere et adresseudtryk når der er sat hak ved 'Anvend kun valgte objekter',

### ISSUE THREAD

**Issue by MieWinstrup** [Septima/qgis-addresstoolsdk#2]
Når der er sat hak ved 'Anvend kun valgte objekter', så populeres drop-down ved 'Adresse-udtryk' ikke. 

![image](https://user-images.githubusercontent.com/16276034/79455923-e8ec5300-7fed-11ea-8d6a-d8f3d8c09d61.png).

Og ingen felter vises i expression-builder:
![image](https://user-images.githubusercontent.com/16276034/79456274-7f207900-7fee-11ea-8866-6f09be210662.png)

Fjernes hakket ved 'Anvend kun valgte objekter' så populeres drop-down (og det virker således efter hensigten):
![image](https://user-images.githubusercontent.com/16276034/79456363-ac6d2700-7fee-11ea-94a9-5653c5c58ba2.png)

**Comment by MFuglsang** [comment]
Denne kan jeg ikke reproducere længere @MieWinstrup  ;-)

**Comment by MieWinstrup** [comment]
Ja, det ser ikke ud til at være et problem længere. Dette issue lukkes.
